--- git status ---
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/config/unitree_go2_amp/rough_env_cfg.py
	modified:   source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py
	modified:   source/rl_lab_scrips/joystick.py
	modified:   source/rl_lab_scrips/train.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/config/unitree_go2_amp/rough_env_cfg.py b/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/config/unitree_go2_amp/rough_env_cfg.py
index 603fb53..19cea9d 100644
--- a/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/config/unitree_go2_amp/rough_env_cfg.py
+++ b/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/config/unitree_go2_amp/rough_env_cfg.py
@@ -60,7 +60,7 @@ class UnitreeA1AmpRoughEnvCfg(LocomotionVelocityRoughEnvCfg):
         self.events.physics_material = None
         self.events.randomize_actuator_gains = None
         self.events.randomize_joint_parameters = None
-        self.events.push_robot = None   
+        #self.events.push_robot = None   
         self.events.add_base_mass.params["mass_distribution_params"] = (-1.0, 3.0)
         self.events.add_base_mass.params["asset_cfg"].body_names = "base"
         self.events.base_external_force_torque.params["asset_cfg"].body_names = "base"
@@ -99,7 +99,7 @@ class UnitreeA1AmpRoughEnvCfg(LocomotionVelocityRoughEnvCfg):
         self.rewards.joint_vel_limits.weight = 0
 
         # Action penalties
-        #self.rewards.action_rate_l2.weight = 0
+        self.rewards.action_rate_l2.weight = 0
         # UNUESD self.rewards.action_l2.weight = 0.0
 
         # Contact sensor
@@ -130,14 +130,13 @@ class UnitreeA1AmpRoughEnvCfg(LocomotionVelocityRoughEnvCfg):
         # ------------------------------Commands------------------------------
         self.commands.base_velocity.ranges.lin_vel_x = (-1.0, 2.0)
         self.commands.base_velocity.ranges.lin_vel_y = (-0.3, 0.3)
-        self.commands.base_velocity.ranges.ang_vel_z = (-1.5, 1.5)
+        self.commands.base_velocity.ranges.ang_vel_z = (-3, 3)
         # ------------------------------AMP------------------------------
         self.urdf_path = "datasets/go2_description/urdf/go2_description.urdf"
         self.ee_names = ["FL_foot", "FR_foot", "RL_foot", "RR_foot"]
         self.base_name = "base"
         self.reference_state_initialization = True
         self.amp_motion_files = "datasets/mocap_motions_go2"
-        self.amp_num_preload_transitions = 5000000
         self.amp_replay_buffer_size = 100000
 
 
diff --git a/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py b/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py
index 52d8f5e..a524fc7 100644
--- a/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py
+++ b/source/extensions/isaac.rl_lab/rl_lab/tasks/manager_based/locomotion/velocity/velocity_env_cfg.py
@@ -135,7 +135,7 @@ class CommandsCfg:
 
     base_velocity = mdp.UniformVelocityCommandCfg(
         asset_name="robot",
-        resampling_time_range=(5, 10.0),
+        resampling_time_range=(5.0, 10.0),
         rel_standing_envs=0.02,
         rel_heading_envs=1.0,
         heading_command=True,
@@ -379,7 +379,7 @@ class RewardsCfg:
         weight=0,
         params={"asset_cfg": SceneEntityCfg("robot", joint_names=".*")},
     )
-    action_rate_l2 = RewTerm(func=mdp.action_rate_l2, weight=-0.01)
+    action_rate_l2 = RewTerm(func=mdp.action_rate_l2, weight=0)
     # UNUESD action_l2
 
     # Contact sensor
@@ -443,7 +443,7 @@ class RewardsCfg:
     # )
     feet_slide = RewTerm(
         func=mdp.feet_slide,
-        weight=-0.25,
+        weight=0,
         params={
             "sensor_cfg": SceneEntityCfg("contact_forces", body_names=".*ankle_link"),
             "asset_cfg": SceneEntityCfg("robot", body_names=".*ankle_link"),
diff --git a/source/rl_lab_scrips/joystick.py b/source/rl_lab_scrips/joystick.py
index 454e9ad..bd22388 100644
--- a/source/rl_lab_scrips/joystick.py
+++ b/source/rl_lab_scrips/joystick.py
@@ -148,9 +148,9 @@ def main():
             
             readings = joystick.advance()
             if readings[0] < 0:
-                readings[0] = 0.5 * readings[0]
+                readings[0] =  readings[0]
             else :
-                readings[0] = 1*readings[0]
+                readings[0] = 2*readings[0]
             readings[1] = 0.15 * readings[1]
             readings[2] = -1.5*readings[2]
             readings_tensor = torch.from_numpy(readings*2).cuda()  #  NumPy  PyTorch  GPU
diff --git a/source/rl_lab_scrips/train.py b/source/rl_lab_scrips/train.py
index 0680884..3cc6a23 100644
--- a/source/rl_lab_scrips/train.py
+++ b/source/rl_lab_scrips/train.py
@@ -22,7 +22,7 @@ parser.add_argument("--video", action="store_true", default=False, help="Record
 parser.add_argument("--video_length", type=int, default=200, help="Length of the recorded video (in steps).")
 parser.add_argument("--video_interval", type=int, default=2000, help="Interval between video recordings (in steps).")
 parser.add_argument("--num_envs", type=int, default=None, help="Number of environments to simulate.")
-parser.add_argument("--task", type=str, default="Isaac-Ase-Unitree-go2-v0", help="Name of the task.")
+parser.add_argument("--task", type=str, default="Isaac-Amp-Unitree-go2-v0", help="Name of the task.")
 parser.add_argument("--seed", type=int, default=None, help="Seed used for the environment")
 parser.add_argument("--max_iterations", type=int, default=None, help="RL Policy training iterations.")
 
@@ -39,7 +39,7 @@ if args_cli.video:
     
 ################################set headless mode############################################
 setattr(args_cli, 'headless', True)
-#setattr(args_cli, 'resume', True)
+setattr(args_cli, 'resume', True)
 ################################set headless mode############################################
 
 # clear out sys.argv for Hydra